tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -8449,7 +8449,7 @@ void VerifyAtomicResults(const BYTE *uResults, const BYTE *sResults,
   // Test ADD Operation
   // ADD just sums all the indices. The result should the sum of the highest and lowest indices
   // multiplied by half the number of sums.
-  size_t addResult = (maxIdx)*(maxIdx-1)/2;
+  uint64_t addResult = (maxIdx)*(maxIdx-1)/2;
```

Comment:
This is the one that's technically unnecessary, but it's not a "size" either and it's more consistent to say that results should be 64-bits.